launcher/launcher.cpp

```diff
@@ -278,7 +278,7 @@ void Launcher::restartTimer()
 
 void Launcher::checkDone()
 {
-    if (d->state == Complete
+    if (d->state == InjectorFinished
```

Comment:
This expression now does some redundant checks: Right now it's `a || (b && a)` => Could be simplified to just `a`. Not sure if that's intended.

I'm not sure if this change is fine, that's probably something for @vkrause.